### ISSUE TITLE
report: Hex-Rays Decompiler (part of Related Work) [6h]

### ISSUE THREAD

**Issue by mewmew** [decomp/decomp#45]
Depends on issue #80.

**Comment by mewmew** [comment]
Prerequisite for issue #35.

**Comment by mewmew** [comment]
Merged into the Decompilers section. Closing this issue in favour of the issue of the Decompilers section.
